app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp = command.split(" ")
```

Comment:
No need to use space as arg for split method, it use space by default

Reply:
Ok, fixed)